### PR TITLE
Optimize my API for MongoDB cost and performance

### PR DESCRIPTION
**Pull request by t-khelan**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by t-khelan** [comment]
@StefArroyo

File: Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook
Comment (gahl-levy) [inline Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:1107]:
```suggestion
        "json": "# Step 4. Enable Log Analytics for additional troubleshooting\r\n\r\nUse Azure Diagnostic Logs to identify which requests are returning 50s and how many RUs they consumed. This sample query aggregates at the minute level."
```

Comment (gahl-levy) [reply Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:1107]: I believe this should be 50 since we're looking for timeout errors?

File: Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook
Comment (gahl-levy) [inline Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:1241]: This is describing the SQL API. Seems out of place in this workbook. Should we remove this or edit this for the Mongo API?

Comment (gahl-levy) [reply Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:1241]: I'd suggest removing this section to unblock us.

File: Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook
Comment (gahl-levy) [inline Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:7]: I'd suggest changing the title to "Optimize my API for MongoDB cost and performance"

Comment (gahl-levy) [reply Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook:7]: Since customers will likely come from a higher level mindset rather than questioning a single graph.

**Review by gahl-levy — CHANGES_REQUESTED** [review]
A few small changes

**Review by gahl-levy — APPROVED** [review]

**Review by StefArroyo — APPROVED** [review]

File: gallery/workbook/microsoft.documentdb-databaseaccounts.json
Comment (gardnerjr) [inline gallery/workbook/microsoft.documentdb-databaseaccounts.json:15]: you should add an entry to the `CODEOWNERS` file in the gallery section for this file, so that the cosmosdb team owns this gallery file instead of the workbooks team?

File: gallery/workbook/microsoft.documentdb-databaseaccounts.json
Comment (gardnerjr) [inline gallery/workbook/microsoft.documentdb-databaseaccounts.json:17]: should this long text be the description? the names are usually short things with a longer description?

File: CODEOWNERS
Comment (t-khelan) [inline CODEOWNERS:76]: @gardnerjr is this the correct syntax?

File: gallery/workbook/microsoft.documentdb-databaseaccounts.json
Comment (t-khelan) [reply gallery/workbook/microsoft.documentdb-databaseaccounts.json:15]: done

File: gallery/workbook/microsoft.documentdb-databaseaccounts.json
Comment (t-khelan) [reply gallery/workbook/microsoft.documentdb-databaseaccounts.json:17]: i believe this is the name of the workbook

File: CODEOWNERS
Comment (gardnerjr) [reply CODEOWNERS:76]: this one you don't need to do, because the line above with ** already covers everything inside workbooks/cosmosdb folder.

the one i'm talking about is further down in this file, there should be lines like

```
/gallery/workbook/microsoft.documentdb-databaseaccounts.json @microsoft/azurecosmosdb-workbooks
/gallery/cosmosdb-insights/** @microsoft/azurecosmosdb-workbooks
```
in the section that contains gallery files/folders.  that change would have cosmosdb team take ownership of the specific gallery files your team owns (and one modified in this PR)

**Review by gardnerjr — APPROVED** [review]

**Pull request merged**
